src/core/api/playback_observer.ts

```diff
@@ -207,6 +199,10 @@ export default class PlaybackObserver {
    * @returns {Observable}
    */
   private _createInnerObservable() : Observable<IPlaybackObservation> {
+
+    const { SAMPLING_INTERVAL_MEDIASOURCE,
+            SAMPLING_INTERVAL_LOW_LATENCY,
+            SAMPLING_INTERVAL_NO_MEDIASOURCE } = config.getCurrent();
```

Comment:
When in that situation (just before a `defer`, I think it makes more sense to get the current config inside the `defer` so that it happens when we need it.

Outside the defer is the code called when the function is called.
Inside the defer is the code called when the observable is subscribed to.

What do you think?

Reply:
Fixed